shared/common-adapters/popup-dialog.desktop.js

```diff
@@ -10,13 +10,13 @@ function stopBubbling (ev) {
   ev.stopPropagation()
 }
 
-export function PopupDialog ({children, onClose, fill}: Props) {
+export function PopupDialog ({children, onClose, fill, styleCover, styleContainer, styleClose, styleClipContainer, allowClipBubbling}: Props) {
   return (
     <EscapeHandler onESC={onClose}>
-      <Box style={styleCover} onClick={onClose}>
-        <Box style={{...styleContainer, ...(fill ? styleContainerFill : null)}}>
-          <Icon type='iconfont-close' style={styleClose} />
-          <Box style={styleClipContainer} onClick={stopBubbling}>
+      <Box style={{...coverStyle, ...styleCover}} onClick={onClose}>
+        <Box style={{...containerStyle, ...(fill ? containerFillStyle : null), ...styleContainer}}>
+          <Icon type='iconfont-close' style={{...closeStyle, ...styleClose}} />
+          <Box style={{...clipContainerStyle, ...styleClipContainer}} onClick={allowClipBubbling ? undefined : stopBubbling}>
```

Comment:
If clicks are allowed to bubble out, they'll trigger the `onClose` of the parent. This seems to happen in the edit popup when clicking on the text to move the cursor. Is this intended?